src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        final int arrLen = 6;
```

Comment:
please rename in more self explained manner e.g. `figuresCount`. also please do not cut the words in the names.